packages/myst-compat/src/index.ts

```diff
@@ -0,0 +1,21 @@
+import { downgrade } from './downgrade/index.js';
+import { upgrade } from './upgrade/index.js';
+import type { IFile } from './types/index.js';
+
+export function makeCompatible(src: IFile, to: string) {
```

Comment:
Change this to `opts: {to: string }`?
```suggestion
export function migrate(src: IFile, to: string) {
```

Reply:
In meeting the version is required. Can add options as third arg.